packages/modal/src/components/modal/Modal.tsx

```diff
@@ -18,6 +20,8 @@ export const Modal: React.FC<ModalProps> = ({
   draggable = false,
   onRequestClose,
   closeButtonClassName,
+  footer,
+  stickyFooter = true,
```

Comment:
Never default to `true`.
Rename prop to `disableStickyFooter` and default to `false`.

Reply:
Fixed!